src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const sourceObject = {};
+  const declarations = sourceString.split(';');
```

Comment:
Directly splitting the string with ';' can lead to empty entries in the array if there are consecutive semicolons. It would be better to first trim the string and then split it, or use a more robust method to handle potential formatting issues.